audeer/core/io.py

```diff
@@ -338,27 +339,41 @@ def list_dir_names(
         path: typing.Union[str, bytes],
         *,
         basenames: bool = False,
-) -> typing.List:
+        recursive: bool = False,
+) -> typing.List[str]:
     """List of folder names located inside provided path.
 
     Args:
         path: path to directory
         basenames: if ``True`` returns basenames of directories
```

Comment:
I would also change it here like we did for `audeer.list_file_names()`:

```suggestion
        basenames: if ``True`` return relative path in respect to ``path``
```